commands/http.js

```diff
@@ -20,7 +21,7 @@ module.exports = {
 
     //Destructure and get properties ready.
     const taskProps = utils.resolveInputParameters();
-    const { url, method, header, contentType, body, allowUntrustedCerts, outputFilePath } = taskProps;
+    const { url, method, header, contentType, body, allowUntrustedCerts, outputFilePath, errorcodes = "", httperrorretry = 5 } = taskProps;
```

Comment:
One nitpick here. The default for error retries should probably match the existing (or non existent) functionality i.e. be 0. And then changed as needed by end user.

Also, make sure to match with a default in the task template.

Reply:
In case the errorcodes is an empty string the number of retires are ignored.
Will do the change as suggested.